### DISCUSSION TITLE
Double click invocation to bring in fields into Expression Editor

### DISCUSSION THREAD

**Discussion by cygnex** [TabularEditor/TabularEditor3#225] [Ideas]
I am not sure how easy this would be but an idea for a future release could be to be able to double click on a field and it would be brought into the Expression Editor. 

This could speed up development time and could be faster then dragging and dropping content.

**Comment by otykier** [comment]
I believe this is already possible today, as long as you double-click on the icon to the left of the object name. Default behavior in most windows applications when you double click the name is that you want to edit the name.
